replica/table.cc

```diff
@@ -1149,22 +1149,23 @@ void table::start_compaction() {
 }
 
 void table::trigger_compaction() {
-    // Submitting compaction job to compaction manager.
-    do_trigger_compaction(); // see below
+    for (compaction_group_ptr& cg : compaction_groups()) {
+        do_trigger_compaction(*cg);
+    }
```

Comment:
Should this be moved to compaction_group? For example, if an sstable is added to a compaction group, we only need to trigger compaction there.

Reply:
Yes